RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,240 @@
+## Subject Line
+[Blox] Deploy release - v$$blox_version$$
+
+## Activity Details
+This activity is to: Release Blox version v$$blox_version$$ by pushing our code up to the master branch on GitHub, and pushing new Docker images up to Docker Hub.
```

Comment:
I would also be more specific: what docker images? How about "updating blox images in dockerhub"?